sources/packages/web/src/main.ts

```diff
@@ -33,7 +33,7 @@ import FooterButtons from "@/components/generic/FooterButtons.vue";
 import TitleValue from "@/components/generic/TitleValue.vue";
 import StatusInfoLabel from "@/components/generic/StatusInfoLabel.vue";
 
-AppConfigService.shared.init().then(() => {
+await AppConfigService.shared.init().then(() => {
```

Comment:
The await does not make sense in this case because it is not inside an `async` method. Was this a Sonarcloud suggestion?

Reply:
Yes, you are correct. `Await` does not make sense here because it is already being handled in the `.then()`. However, the Sonarcloud was complaining about this issue because the `.then()` was being used without the rejection handler. I have added `.catch()` now and this handles the rejection scenario fixing the sonar complaint.